Cloudflare/audit_logs/ingest/parser.yml

```diff
@@ -0,0 +1,66 @@
+name: cloudflare-audit-logs
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+  - name: set_ecs_fields
+  - name: set_cloudflare_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": '{{json_event.message.When}}'
+      - set:
+          cloud.account.id: '{{json_event.message.OwnerID}}'
+        filter: '{{json_event.message.OwnerID is not in [""]}}'
+      - set:
+          event.category: ["network"]
+          event.dataset: "audit_logs"
+          event.kind: 'event'
+          event.type: ["access"]
+          observer.type: "proxy"
+          observer.vendor: "Cloudflare"
+          source.ip: '{{json_event.message.ActorIP}}'
+          user.email: '{{json_event.message.ActorEmail}}'
+          user.id: '{{json_event.message.ActorID}}'
+
+  set_cloudflare_fields:
+    actions:
+      - set:
+          cloudflare.ActionType: '{{json_event.message.ActionType}}'
```

Comment:
I wonder if `event.action` is not suitable for this information.

Reply:
You're right, I'll change this